src/config.mli

```diff
@@ -25,6 +25,7 @@ type config =
     manitou : bool;
     supervisor : bool;
     wizard : bool;
+    template : bool;
```

Comment:
So this should clearly be renamed into something meaningful like is_XXX since it's a boolean value